e2e/suite1/specs/explore.spec.ts

```diff
@@ -25,6 +25,6 @@ e2e.scenario({
       });
 
     const canvases = e2e().get('canvas');
-    canvases.should('have.length', 2);
+    canvases.should('have.length', 1);
```

Comment:
Wondering/not sure why this needs to be changed?

Reply:
This test was failing because of this. The old graph panel had 2 canvases 🤷‍♂️